release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1058,6 +1076,18 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-web-console-developer-perspective-bug-fixes"]
 ==== Web console (Developer perspective)
 
+* Before this update, the *Git Import* form displayed an error message when an invalid devfile repository (older than devfile v2.2) is entered. With this update, the error message state that devfiles older than v2.2 is not supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046435[*BZ#2046435*])
+
+* Before this update, if the *ConsoleLink CR* (openshift-blog) was not available in the cluster that causes the blog link to be undefined. So while clicking on the blog link did not redirect to the openshift blog. With this update fall back link to https://developers.redhat.com/products/openshift/whats-new is added even if the *ConsoleLink CR* (openshift-blog) is not present in the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050637[BZ#2050637])
+
+* Before this update, the API version for kafka CR got updated that did not support the old version which used to show empty *Bootstrap server* dropdown in *Create Event Source - KafkaSource* form even if it was created. With this update, the updated APIs for Kafka CR supports the old versions and renders the *Bootstrap server* dropdown in *Create Event Source - KafkaSource* form.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2058623[BZ#2058623])
+
+* Before this update, when you use the *Import from Git* form to import a private git repository, the correct import type and a builder image were not identified. Because the secret to fetch the private repository details is not decoded. With this update the *Import from Git* form decodes the secret to fetch the private repository details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053501[BZ#2053501])
```

Comment:
```suggestion
* Before this update, when you use the *Import from Git* form to import a private git repository, the correct import type and a builder image were not identified. Because the secret to fetch the private repository details is not decoded. With this update, the *Import from Git* form decodes the secret to fetch the private repository details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053501[BZ#2053501])
```